### PR TITLE
Move markov commands

### PR DESCRIPTION
**Pull request by skonnov**

### CODE REVIEW

**Comment by skonnov** [comment]
I have 2 proposals which I added in last 3 commits:
1) I don't see much difference between *markovignoredprefix and *markovfilter commands, so I suggest to change *markovfilter permissions and subcommand settings to make it similar to *markovfilter
2) I found that there are no check for ignored_prefix in telegram pipeline, so I suggest to add this check to src/markov.py in Markov::add_string function

**Comment by aobolensk** [comment]
These two proposals make sense to me. Thank you for pointing that out.
1. Yes, these commands have had permission mess. They definitely need to be aligned.
2. Since ignored prefix was not a thing that can be set up (before your commit) it was not used on this backend.

File: src/markov.py [hunk 1]
```diff
@@ -62,6 +62,10 @@ def __init__(self):
     def add_string(self, text: str) -> None:
         if len(text) < self.min_chars or len(text) > self.max_chars:
             return
+        for prefix in self.ignored_prefixes.values():
+            print(prefix)
```
Comment (aobolensk) [inline src/markov.py:66]: Debug print

File: src/cmd/markov.py [hunk 2]
```diff
@@ -180,3 +198,74 @@ async def _inspectmarkov(self, cmd_line: List[str], execution_ctx: ExecutionCont
...
+        if not await Command.check_args_count(execution_ctx, cmd_line, min=1, max=1):
+            return
+        result = ""
+        for index, prefix in bc.markov.ignored_prefixes.items():
+            result += f"{index} -> `{prefix}`\n"
+        await Command.send_message(execution_ctx, result or "No ignored prefixes for Markov model found!")
+
+    async def _delmarkovignoredprefix(self, cmd_line: List[str], execution_ctx: ExecutionContext):
+        """Delete message prefix that should be ignored by Markov model by its index
+    Example: !delquote 0"""
```
Comment (aobolensk) [inline src/cmd/markov.py:260]: Just noticed in on review of your PR. This command has an incorrect example in the docstring. Could you please fix?

**Review by aobolensk — CHANGES_REQUESTED** [review]

File: src/markov.py (hunk 1 above)
Comment (skonnov) [reply src/markov.py]: Removed

File: src/cmd/markov.py (hunk 2 above)
Comment (skonnov) [reply src/cmd/markov.py]: Fixed

**Review by aobolensk — APPROVED** [review]
LGTM!
Thank you very much for your contribution!

**Pull request merged**